test/Sentry.Tests/SentryHttpFailedRequestHandlerTests.cs

```diff
@@ -265,4 +347,62 @@ public void HandleResponse_Hint_Response()
             hint.Items[HintTypes.HttpResponseMessage].Should().Be(response);
         }
     }
+
+    [Fact]
+    public void HandleResponse_ExceptionHasStackTrace()
+    {
+        // Arrange
+        var options = new SentryOptions
+        {
+            CaptureFailedRequests = true
+        };
+        var sut = GetSut(options);
+
+        var response = InternalServerErrorResponse();
+        response.RequestMessage = new HttpRequestMessage(HttpMethod.Get, "http://example.com/api/test");
+
+        // Act
+        SentryEvent @event = null;
+        _hub.CaptureEvent(Arg.Do<SentryEvent>(e => @event = e), hint: Arg.Any<SentryHint>());
+        sut.HandleResponse(response);
+
+        // Assert
+        using (new AssertionScope())
+        {
+            @event.Should().NotBeNull();
+            @event.Exception.Should().NotBeNull();
+            @event.Exception!.StackTrace.Should().NotBeNullOrWhiteSpace();
+        }
+    }
+
+#if NET6_0_OR_GREATER // This test is only valid on .NET 6+ where we can use SetRemoteStackTrace
+    [Fact]
+    public void HandleResponse_StackTraceIncludesCallerContext()
```

Comment:
suggestion: Add a "scenario-part" in the middle of the test name, to be compliant with out `CONTRIBUTING.md`.

Maybe something like

```suggestion
    public void HandleResponse_Capture_StackTraceIncludesCallerContext()
```